File: Ejercicios/06.Corredores listos fuera!/Corredoreslistosfuera!.py
def main ():
    minutos = 1
    segundos = 1
    
    while minutos !=0 or segundos != 0:
        
        while True:
            try:
                minutos,segundos = map(int, input("Ingrese tiempos en mm ss (0 0 para salir) : ").split())
                if minutos < 0 or segundos < 0:
                    print("\n Ha ingresado un valor invalido, debe ser >= 0. Reintente! \n")
                else:
                    break
            except ValueError :
            
             print("\Entrada invalida. Ingrese dos numeros separados por espacio.\n")
    
    if segundos != 0 or minutos != 0:
        tiempo = segundos+(minutos*60)
        velocidad= 1500/tiempo
        print(f"La velocidad promedio es de : {velocidad:.2f} m/s \n")
        
if __name__ == "__main__":
    main()

File: minions/6.Corredores listos fuera!/Corredoreslistosfuera!.c
#include <stdio.h>
#include <stdlib.h>

int main(){
    int minutos,segundos,tiempo;
    float velocidad;

    do{
        do
        {
            printf("Ingrese tiempos en mm ss: ");
            scanf("%d %d",&minutos,&segundos);
            if(minutos<0 || segundos<0) printf("\nha ingresado un valor invalido, debe ser >=0. Reintente!\n ");
        } while (minutos<0 || segundos<0);
        
        if (segundos!=0 || minutos!=0){
            tiempo=segundos+(minutos*60);
            velocidad=1500/(float)tiempo;
            printf("La velocidad promedio es de: %.2f m/s\n",velocidad);
        }    
        
    } while (minutos!=0 || segundos!=0);   

    return EXIT_SUCCESS;
}